### PR TITLE
Add tags for all docs pages (except submodule pages)

### PR DESCRIPTION
**Pull request by jessiemongeon1**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: docs/developer-docs/backend/motoko/counter-tutorial.mdx [hunk 1]
```diff
@@ -1,8 +1,11 @@
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
 
 # 11: Incrementing a natural number
 
+<MarkdownChipRow labels={["Beginner", "Motoko", "Tutorial"]} />
```
Comment (dfx-json) [inline docs/developer-docs/backend/motoko/counter-tutorial.mdx:7]: note to self: the chips should be updated to support screen readers and SEO

File: docs/developer-docs/backend/motoko/define-an-actor.mdx [hunk 2]
```diff
@@ -1,8 +1,12 @@
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
 
 # 14: Querying using an actor
 
+<MarkdownChipRow labels={["Intermediate", "Motoko", "Tutorial"]} />
```
Comment (dfx-json) [inline docs/developer-docs/backend/motoko/define-an-actor.mdx:8]: for SEO, I think these same tags should be added into the frontmatter as keywords.

**Review by dfx-json — COMMENTED** [review]
Approved, but made a small note about supporting SEO. These tags ideally should make it into the front matter as keywords. Up to you if you'd like to include in this PR or raise a separate PR.

I will also follow up separately to support screen readers in the chip component

**Review by dfx-json — APPROVED** [review]

File: docs/developer-docs/backend/motoko/multiple-actors.mdx [hunk 3]
```diff
@@ -1,8 +1,11 @@
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
 
 # 15: Using multiple actors
 
+<MarkdownChipRow labels={["Intermediate", "Motoko", "Tutorial"]} />
```
[the remark this answers is not in the document]
Comment (jessiemongeon1) [reply docs/developer-docs/backend/motoko/multiple-actors.mdx:7]: Sounds good, I will create a separate PR to add SEO keywords to all docs pages.

**Pull request merged**